fixtures/discount_code.json

```diff
@@ -1,10 +1,10 @@
 {
-    "discount_code": {
-      "id": 1054381139,
-      "price_rule_id": 507328175,
-      "code": "SUMMERSALE10OFF",
-      "usage_count": 0,
-      "created_at": "2018-07-05T13:04:26-04:00",
-      "updated_at": "2018-07-05T13:04:26-04:00"
-    }
-  }
\ No newline at end of file
+  "discount_code": {
+     "id": 1054381139,
+     "price_rule_id": 507328175,
+     "code": "SUMMERSALE10OFF",
+     "usage_count": 0,
+     "created_at": "2018-07-05T13:04:26-04:00",
+     "updated_at": "2018-07-05T13:04:26-04:00"
+  }
+}
```

Comment:
Hey @MarinX! would you be able to add a newline after line 10 here?

Reply:
Hey @andrewhoff . Its added. I thought it was a problem on formatting the JSON.
May I ask why newline is needed (carriage return?) since some of the files also does not have new lines.
Just want to be consistent with future PR's :)

Reply:
all good! I think it's an POSIX compliance thing that git complains about.

https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file
https://robots.thoughtbot.com/no-newline-at-end-of-file